stacks/port_tcp.go

```diff
@@ -9,13 +9,23 @@ import (
 	"github.com/soypat/seqs/eth"
 )
 
+// <unhelpful and misleading !>
 // tcphandler represents a user provided function for handling incoming TCP packets on a port.
 // Incoming data is sent inside the `pkt` TCPPacket argument when pkt.HasPacket returns true.
 // Outgoing data is stored into the `response` byte slice. The function must return the number of
 // bytes written to `response` and an error.
-//
+// </unhelpful and misleading !>
+
+// <suggeted replacement>
+// tcphandler represents a user provided function for handling incoming TCP packets on a port.
+// Incoming data is passed in a 'pkt' to the recv function which is invoked whenever data arrives (by RecvEth)
+// Outgoing data is written into the `dst` byte slice (from the tx ring buffer). The function must return the number of
+// bytes written to `response` and an error.
+// TCPConn provides an implemntation of this interface - note .send is ONLY called by HandleEth
```

Comment:
No need for xml tags, github PRs show previous and new changes. Your changes reflect the latest API, so should be more helpful than a description of a previous API.